modules/custom/openy_activity_finder/openy_activity_finder.libraries.yml

```diff
@@ -10,21 +10,28 @@ openy_activity_finder:
     - openy_repeat/openy_repeat.vue
     - openy_repeat/openy_repeat.vue_router
     - openy_repeat/openy_repeat.vue.table
-    - openy_repeat/open_repeat.moment
-    - openy_repeat/open_repeat.bootstrap-datepicker
+    - openy_repeat/openy_repeat.moment
+    - openy_repeat/openy_repeat.bootstrap-datepicker
```

Comment:
I think we can remove bootstrap-datepicker from programs search. We do not have date picker anywhere.

Reply:
Sounds good.